features/structured-clone.yml

```diff
@@ -1,3 +1,4 @@
 name: structuredClone()
 description: The `structuredClone()` global method creates a deep copy of an object. Values that cannot be cloned can instead be transferred, making the original value no longer usable.
 spec: https://html.spec.whatwg.org/multipage/structured-data.html#structured-cloning
+group: dom
```

Comment:
Actually no, `structuredClone()` handles arbitrary JS values. And it can't clone elements, it seems from my testing.